.github/workflows/purge.yml

```diff
@@ -0,0 +1,24 @@
+name: Purge Cache (jsDelivr)
+
+on:
+  push:
+    paths:
+      - 'packages/synapse-interface/public/pauses/v1/**'
+
+jobs:
+  purge-cache:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
```

Comment:
not needed

Reply:
removed in https://github.com/synapsecns/sanguine/pull/2857/commits/ee521e1c39219ac5b072e6a31ad6d7f58a3ec456